x-pack/plugins/aiops/public/components/change_point_detection/change_points_table.tsx

```diff
@@ -0,0 +1,364 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { type EuiBasicTableColumn, EuiInMemoryTable, EuiBadge, EuiEmptyPrompt } from '@elastic/eui';
+import React, { type FC, useMemo } from 'react';
+import { i18n } from '@kbn/i18n';
+import { useTimeRangeUpdates } from '@kbn/ml-date-picker';
+import { FilterStateStore } from '@kbn/es-query';
+import { TypedLensByValueInput } from '@kbn/lens-plugin/public';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiTableSelectionType } from '@elastic/eui/src/components/basic_table/table_types';
+import { useDataSource } from '../../hooks/use_data_source';
+import { fnOperationTypeMapping } from './constants';
+import {
+  type ChangePointAnnotation,
+  FieldConfig,
+  SelectedChangePoint,
+  useChangePointDetectionContext,
+} from './change_point_detection_context';
+import { type ChartComponentProps } from './chart_component';
+import { useAiopsAppContext } from '../../hooks/use_aiops_app_context';
+
+export interface ChangePointsTableProps {
+  annotations: ChangePointAnnotation[];
+  fieldConfig: FieldConfig;
+  isLoading: boolean;
+  onSelectionChange: (update: SelectedChangePoint[]) => void;
+}
+
+export const ChangePointsTable: FC<ChangePointsTableProps> = ({
+  isLoading,
+  annotations,
+  fieldConfig,
+  onSelectionChange,
+}) => {
+  const { fieldFormats } = useAiopsAppContext();
+
+  const dateFormatter = useMemo(() => fieldFormats.deserialize({ id: 'date' }), [fieldFormats]);
+
+  const sorting = {
+    sort: {
+      field: 'p_value',
+      direction: 'asc' as const,
```

Comment:
Would be useful to keep the removed comment from the other file here for text `// Lower p_value indicates a bigger change point, hence the acs sorting`

Reply:
good point, I'll also add a helper text to the column header for the user